docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TreeFeaturization/BinaryClassificationFeaturization.ttinclude

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+<# if (TrainerOptions != null) { #>
+<#=OptionsInclude#>
+<# } #>
+
+namespace Samples.Dynamic.Transforms.TreeFeaturization
+{
+    public static class <#=ClassName#>
+    {<#=Comments#>
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of data points to be transformed.
+            var dataPoints = GenerateRandomDataPoints(1000).ToList();
+
+            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            var dataView = mlContext.Data.LoadFromEnumerable(dataPoints);
+<# if (CacheData) { #>
+
+            // ML.NET doesn't cache data set by default. Therefore, if one reads a data set from a file and accesses it many times,
+            // it can be slow due to expensive featurization and disk operations. When the considered data can fit into memory,
+            // a solution is to cache the data in memory. Caching is especially helpful when working with iterative algorithms 
+            // which needs many data passes.
+            dataView = mlContext.Data.Cache(dataView);
+<# } #>
+
+            // Define input and output columns of tree-based featurizer.
+            string labelColumnName = nameof(DataPoint.Label);
+            string featureColumnName = nameof(DataPoint.Features);
+            string treesColumnName = nameof(TransformedDataPoint.Trees);
+            string leavesColumnName = nameof(TransformedDataPoint.Leaves);
+            string pathsColumnName = nameof(TransformedDataPoint.Paths);
+
+			// Define the configuration of the trainer used to train a tree-based model.
```

Comment:
>			 [](start = 0, length = 3)

nit: you had some tab heresy in your `tt` files.

Reply:
Removed. Thanks.

---
In reply to: [291365453](https://github.com/dotnet/machinelearning/pull/3812#discussion_r291365453) [](ancestors = 291365453)